tailormap-components/projects/core/src/lib/analysis/create-layer-styling/create-layer-styling.component.ts

```diff
@@ -113,7 +117,11 @@ export class CreateLayerStylingComponent implements OnInit, OnDestroy {
   }
 
   public getStyleLabel(style: UserLayerStyleModel) {
-    return StyleHelper.getStyleLabel(style);
+    const label = [ StyleHelper.getStyleLabel(style) ];
+    if (style.featureCount) {
+      label.push(`(${style.featureCount} objecten)`);
```

Comment:
Is het een ide om "objecten" weg te halen: wordt vaak afgebroken, en ik denk dat het duidelijk is wat het getal erachter betekent